classes/__tests__/vehicle.test.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const VehicleConstructor = require('../vehicle-constructor.js');
+const {Car, Motorcycle } = require('../vehicle-class.js');
+
+let types = ['Constructor', 'Class'];
+
+describe('Vehicles', () => {
+
+  describe('Car', () => {
+    
+    function getCar(type) {
+      switch(type) {
+      case 'Constructor':
+        return new VehicleConstructor.Car('foo');
+        case 'Class':
+        return new Car('Toyota');
```

Comment:
for line 17 you can do the same thing as line 15 but for classes.(this is a suggestion and don't have to take it if you don't want to)